server.go

```diff
@@ -341,6 +341,17 @@ func (sp *serverPeer) addBanScore(persistent, transient uint32, reason string) {
 	}
 }
 
+// OnVerAck is invoked when a peer receives a verack bitcoin message.
+func (sp *serverPeer) OnVerAck(p *peer.Peer, msg *wire.MsgVerAck) {
```

Comment:
I'd rather leave this out completely until it is ready to be added.  I prefer to avoid big `TODO` blocks laying around the code base in general as issues are a better medium for that since they allow discussion.